modules/networking/private_dns_resolvers_inbound_endpoints/output.tf

```
@@ -0,0 +1,3 @@
+output "id" {
+  value = azurerm_private_dns_resolver_inbound_endpoint.pvt_dns_resolver_inbound_endpoint.id
+}
```

add the private_ip_address as an output variable. check if the following is working

```suggestion
}

output "private_ip_address" {
  value = azurerm_private_dns_resolver_inbound_endpoint.pvt_dns_resolver_inbound_endpoint.ip_configurations[0].private_ip_address
}
```